.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Buat laporan bug untuk membantu kami lebih baik
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Versi**
+Tuliskan versi *library* yang digunakan, contoh : v1.0.2.
+
+**Server**  
+*Local*, *Staging* atau *Production*.
+
+**Browser and Device**  
+Sebutkan *browser* dan *device* apa yang digunakan.
+
+**How to reproduce**  
+Tuliskan langkah - langkah yang perlu dilakukan supaya issue muncul: 
+1.  Pergi ke halaman '...'
+2.  Klik  '....'
+3.  *Scroll*  '....'
+4.  Muncul *error*
+
+**Expected result**  
+Tuliskan harapan yang diinginkan dari proses ini.
+
+**Actual result**  
+Tuliskan proses bug yang terjadi.
+
+**Attachment**
+Lampirkan *screentshots* atau *video* bug yang terjadi.
```

Comment:
typo: _`screentshots`_ -> _`screenshots`_

Reply:
okay thanks, i will fix it

Reply:
Fixed at https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/91/commits/9c070e7c8f1d8c992d0f121c5e33995774ccef09